Tasks/FtpUpload/task.json

```diff
@@ -87,6 +87,15 @@
         "required": true,
         "helpMarkDown": "If selected, the relative local directory structure is recreated under the remote directory where files are uploaded.  Otherwise, files are uploaded directly to the remote directory without creating additional subdirectories.<p>For example, suppose your source folder is: <b>`/home/user/source/`</b> and contains the file: <b>`foo/bar/foobar.txt`</b>, and your remote directory is: <b>`/uploads/`</b>.<br>If selected, the file is uploaded to: <b>`/uploads/foo/bar/foobar.txt`</b>.  Otherwise, to: <b>`/uploads/foobar.txt`</b>.",
         "groupName": "advanced"
+    },
+    {
+        "name": "rejectUnauthorized",
+        "type": "boolean",
+        "label": "Reject unauthorized SSL certificates",
+        "defaultValue": "true",
+        "required": true,
+        "helpMarkDown": "If selected, when connecting in secure mode with ftps://, the ftp server certificate is verified with several well known root certificate authorities (e.g. VeriSign).  This option protects against invalid server authentication certificates which may be expired or revoked (e.g. if they were compromized).  Disabling this option should be done with care, but is necessary if using self-signed certificates.",
```

Comment:
"the ftp server certificate" → "the FTP server certificate"
"VeriSign" → "Verisign"
"compromized" → "compromised"